guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -42,6 +42,16 @@ For information on backups, see {AdministeringDocURL}Backing_Up_Server_and_Proxy
 
 ifdef::katello,satellite[]
 +
+. Confirm the certificate tar file name on existing {smart-proxy-context} server:
++
+[options="nowrap" subs="attributes"]
+----
+# grep tar_file /etc/foreman-installer/scenarios.d/capsule-answers.yaml
+tar_file: "/root/myproxy.example.com-certs.tar"
+----
++
+The same name `myproxy.example.com-certs.tar` should be used to regenerate certificates on the {Project} server.
```

Comment:
```suggestion
Use the same name `myproxy.example.com-certs.tar` to regenerate certificates on {ProjectServer}.
```

See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L81

We should also substitue the "myproxy.example.com" FQDN with an attribute.

Please also reword "should be used" to "use".

Reply:
> See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L81
> 
> We should also substitue the "myproxy.example.com" FQDN with an attribute.
No idea which attribute should be used here. I checked  all the attribute files, but I did not get any attribute for myproxy. 
> 
> Please also reword "should be used" to "use".
I think this is no longer required after changing sentence to what you suggested earlier. The final sentence sounds like:
. Use the same name `myproxy.example.com-certs.tar` to regenerate certificates on {ProjectServer}.
Kindly inform if I am missing something.

Reply:
Please also drop this line.

Reply:
Removed.